test/e2e/storage/testsuites/base.go

```diff
@@ -286,10 +287,14 @@ func (r *genericVolumeTestResource) cleanupResource() {
 			}
 			if r.pvc != nil {
 				err := e2epv.DeletePersistentVolumeClaim(f.ClientSet, r.pvc.Name, f.Namespace.Name)
-				framework.ExpectNoError(err, "Failed to delete PVC %v", r.pvc.Name)
+				if err != nil {
+					return errors.Wrapf(err, "Failed to delete PVC %v", r.pvc.Name)
```

Comment:
@msau42 , do you think we should use 
```
err = errors.Wrapf(err, "Failed to delete PVC %v", r.pvc.Name)
```
do not return ? deleteStorageClass should also do not return?

Reply:
I don't know how likely it is that cleaning up will succeed if there are multiple failures. There's also the general removal of the test's namespace that'll delete objects.

But it doesn't hurt to try each delete call and then return an aggregated error, so the current version of `cleanupResource` looks good to me.

You need to rebase, though.

Reply:
I have updated the PR. Thanks

Reply:
GitHub is still showing a conflict for `test/e2e/storage/testsuites/subpath.go`.

Reply:
sorry, conflicts happened since my last commit.

updated now.  PTAL, thanks